tutorial/README.rst

```diff
@@ -105,16 +114,25 @@ uses it to illustrate a range of framework features.
       #. Add a renewable-resource supply curve for the wind power plant,
          (:tut:`westeros/westeros_renewable_resource.ipynb`).
 
-   #. Variablity in energy supply and demand by adding sub-annual time steps,
+   #. Variability in energy supply and demand by adding sub-annual time steps,
```

Comment:
Possibly add list of tutorials below. Currently there is only one tutorial, but more will follow.